StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/FormSpec/FormSpec.swift

```diff
@@ -229,6 +234,27 @@ extension FormSpec {
         /// The list of countries to be displayed for this component
         let allowedCountryCodes: [String]?
     }
+
+    struct PlaceholderSpec: Decodable, Equatable {
+        let field: PlaceholderField
+
+        enum CodingKeys: String, CodingKey {
+            case field = "for"
+        }
+
+        enum PlaceholderField: String, Decodable, Equatable {
+            case name
+            case email
+            case phone
+            case billingAddress = "billing_address"
+            case billingAddressWithoutCountry = "billing_address_without_country"
```

Comment:
It is [already set](https://github.com/stripe/stripe-ios/blob/3edcd2f3/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/FormSpec/FormSpecProvider.swift#L31) to `.convertFromSnakeCase` but it isn't working, I assume because of the custom `init(from decoder:)` implementation.